src/merkledag.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = new Buffer(`
```

Comment:
Is this the only `fs` that required to be brfs'ified in js-ipfs-api? brfs'ified is a standard process in transpiling code to the browser anyway. This seems to me more of a way to postpone getting proper documentation that explains how to do this. Which we have to write anyway.